packages/app/src/update/requests/get-release.request.ts

```diff
@@ -22,11 +20,11 @@ const productsMapper = {
   [DeviceType.MuditaHarmony]: Product.BellHybrid,
 }
 
-const getAllReleases = async (
+export const getRelease = async (
```

Comment:
why did you change the name when the structure response is the same? 🤔 -> `AllReleasesResponse`

Reply:
Fixed